src/pages/CreateFormPage.js

```diff
@@ -22,23 +26,31 @@ const CreateFormPage = () => {
   return (
     <>
       <Header>
-        <Title>Create new form</Title>
+        <Title>
+          <FormattedMessage id="nav.create-form" />
+        </Title>
       </Header>
       <Formik
-        initialValues={{ name: '', category: '' }}
-        onSubmit={useCallback(values => alert(JSON.stringify(values)), [])}
+        initialValues={{ name: '', category: formCategories.length && formCategories[0].id }}
+        validationSchema={Yup.object().shape({
```

Comment:
why are you using Yup here? validate js not enough?

Reply:
Because Formik docs recommend it, and I was planning to get rid of validate js with redux-form

Reply:
Maybe you can delete the `validate js` dependency in another PR :grin:

Reply:
Sure, I haven't dealt with it yet, because I have to refactor react-redux-base login logic, because it uses redux-form and validate.js